.circleci/config.yml

```diff
@@ -33,6 +33,12 @@ jobs:
       - run:
           name: Compile tests
           command: ./gradlew compileTestJava
+      - run:
+          name: '[TEMP] Upgrade git-machete CLI'
```

Comment:
What is to be done to remove that? We need to rebuild and push the docker image, right?

Reply:
![image](https://user-images.githubusercontent.com/3383210/93464236-26c90c80-f8e9-11ea-9d35-ce94e084a386.png)

Reply:
Image regenerated as `gitmachete/intellij-plugin-ci:2.0.0`